app/utils/ipfs.js

```diff
@@ -10,7 +10,18 @@ export async function saveText(text) {
   return saveResult[0].hash;
 }
 
+export async function saveFile(file) {
+  const buf = Buffer.from(file);
+  const saveResult = await getIpfsApi().add(buf);
+  return saveResult[0].hash;
+}
+
 export async function getText(hash) {
   const getResult = await getIpfsApi().get(hash);
   return getResult[0].content.toString('utf8');
 }
+
+export async function getFile(hash) {
```

Comment:
Let's discuss this one.
I think knowing hash of the document it is easy to build URL without call to .get(hash) method. URL of file in IPFS is http://localhost/ipfs/<hash>. We can probably build that URL just by concatination of strings.